### ISSUE TITLE
An issue with overlapping earrings and bows

### ISSUE THREAD

**Issue by Deli-Cake** [Monika-After-Story/MonikaModDev#9903]
The bow somehow covers the earring.
![screenshot0079](https://user-images.githubusercontent.com/104679049/218777758-a7ccb643-3328-4738-812d-e91c256f62a1.png)

**Comment by TooMuchInvigilation** [comment]
Good catch. Speaking of overlapping, what about her eyes that are in front of her hair? Is it intentional? I can understand her eyebrows being visible like that, but I don't think slightly covering her eyes would look bad. Eventually, I think slightly moving a few hair strands around would be viable.

**Comment by Booplicate** [comment]
That's how it was in DDLC and is common for anime styles.
